client/js/components/StatsLeaderboardCell.vue

```diff
@@ -154,6 +154,12 @@ export default {
       const winRate = Math.floor((this.wins / this.weekCount) * 100);
       return `${winRate}%`;
     },
+    losses() {
+      return this.weekCount - this.wins;
+    },
+    winRateText() {
+      return `(${this.wins} Won, ${this.losses} Lost, ${this.weekCount} Total)`;
+    },
```

Comment:
Let's include the WinRatePercentage in this and then the template above can just display the winRateText

Reply:
Done